partitura/musicanalysis/meter.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Meter numerator, Beat, and Tempo estimation.
+
+Implementation adapted from Jakob Woegerbauer 
+based on a model published by Simon Dixon.[1]
+
+References
+----------
+
+.. [1] Simon Dixon (2001), Automatic extraction of 
+    tempo and beat from expressive performances.
+    Journal of New Music Research, 30(1):39–58
+"""
+
+import warnings
+import numpy as np
+# import scipy.spatial.distance as distance
+# from scipy.interpolate import interp1d
+
+from partitura.utils import get_time_units_from_note_array, ensure_notearray, add_field
+
+
+#__all__ = ["estimate_tonaltension"]
```

Comment:
Add __all__ = ["estimate_time"] on this line for the function to be included to the documentationin ReadTheDocs.

Reply:
this is in the ```__init__.py``` of musicanalysis.

Reply:
seems to work, no?

Reply:
I'm not sure, let me compile the ReadTheDocs Locally and I will let you know.